### PR TITLE
MSOMS : Forcefully pull the image,  accept public slave, get hostname…

### PR DESCRIPTION
**Pull request by amitsara**
Made following changes:
    1) Forcefully pull docker imager
    2) Removed container name parameter.
    3) sharing /etc/hostname to get the computer name inside container.
    4) Added resource role

### CODE REVIEW

File: repo/packages/M/msoms/0/marathon.json.mustache [hunk 1]
```diff
@@ -27,14 +32,11 @@
       "privileged": true,
       "parameters": [
         {
```
Comment (ryadav88) [inline repo/packages/M/msoms/0/marathon.json.mustache]: never remove anything from a version. Please create a new version of msoms package and add your changes

Comment (amitsara) [inline repo/packages/M/msoms/0/marathon.json.mustache]: Thanks Ravi, I have created a new version of the package.

File: repo/packages/M/msoms/0/marathon.json.mustache [hunk 2]
```diff
@@ -60,4 +60,4 @@
       "ignoreHttp1xx": false
     }
   ]
-}
+}
```
Comment (amitsara) [inline repo/packages/M/msoms/0/marathon.json.mustache]: Reverted the file completely.

File: repo/packages/M/msoms/1/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+ "packagingVersion": "3.0",
+ "name": "msoms",
+ "version": "1.0.1",
+ "scm": "git@github.com:Microsoft/OMS-docker.git",
+ "maintainer": "omscontainers@microsoft.com",
+ "description": "Microsoft Operations Management Suite(MS OMS) – Container Solution provides real-time analytics for operational data (Performance, Inventory) and monitoring for Docker Containers.",
```
Comment (ryadav88) [inline repo/packages/M/msoms/1/package.json]: once you add the `example`, add a link at the end of `description`. 
Ex: ```Installation documentation: https://github.com/dcos/examples/tree/master/1.8/msoms```

Comment (amitsara) [inline repo/packages/M/msoms/1/package.json]: I will certainly add an example in a separate pull request.
This merge is urgent, we have to check in this for one of our customer. 
Please merge this change if it looks good to you.

Comment (ryadav88) [inline repo/packages/M/msoms/1/package.json]: yes, merging this one. Will create an issue for the example in the other repo and assign it to you. thanks!

**Pull request merged**